# Contains the robot who sent the message, and the zone that has been completed.
uint8  robot_id
uint8  zone_id

